src/grt/src/GlobalRouter.cpp

```diff
@@ -1132,8 +1130,8 @@ void GlobalRouter::computeUserGlobalAdjustments(int min_routing_layer,
     return;
 
   for (int l = min_routing_layer; l <= max_routing_layer; l++) {
-    if (adjustments_[l] == 0) {
-      adjustments_[l] = adjustment_;
+    if (db_->getTech()->findRoutingLayer(l)->getLayerAdjustment() == 0.0) {
+      db_->getTech()->findRoutingLayer(l)->setLayerAdjustment(adjustment_);
```

Comment:
@maliberty How much null checking do we need to do on such lines. Is it guaranteed that all these pointers will exist i.e. because we checked it elsewhere?

Reply:
We should have verified the min/max routing layers when they were set.  I don't think we need to check here.

Reply:
Actually, we get multiple test fails in GRT if this condition is removed. If it is kept, all tests from GRT pass.